woocommerce/woocommerce
# Update wcpay suggestion UI in payment task

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33598.

This PR updates the wcpay suggestion UI in the payment task, and adds `<WCPayBanner />` and `<WCPayBenefits />` to `@woocommerce/onboarding` for the changes.

Before:

![Screen Shot 2022-07-04 at 16 02 26](https://user-images.githubusercontent.com/4344253/177110098-4e970c43-4192-4eba-a03c-7471f00352e3.png)


After:

![Screen Shot 2022-07-04 at 15 48 42](https://user-images.githubusercontent.com/4344253/177109729-f3220236-a9b1-4f9e-a8a9-bae760dcba22.png)

I've also confirmed this change doesn't affect the banner in setting page:

![Screen Shot 2022-07-04 at 16 01 51](https://user-images.githubusercontent.com/4344253/177111212-134c10b1-2a7d-4db8-842b-44483f0989d0.png)


### How to test the changes in this Pull Request:

1. Onboard to WooCommerce from a WCPay eligible country BUT during onboarding opt out from installing WCPay
2. Go to **Woocmmerce > Home**
3. Go to payment task
4. Observe that the UI matches the design (e428PXC0QyEwYqtX48lLq0-fi-12830%3A93773).


### Other information:




### FOR PR REVIEWER ONLY:
